assets/js/components/settings/SettingsApp.stories.js

```diff
@@ -16,38 +16,20 @@
  * limitations under the License.
  */
 
-/**
- * WordPress dependencies
- */
-import { __ } from '@wordpress/i18n';
-
 /**
  * Internal dependencies
  */
-import { Tab, TabBar } from 'googlesitekit-components';
 import Layout from '../layout/Layout';
 import SettingsApp from './SettingsApp';
+import { VIEW_CONTEXT_MAIN_DASHBOARD } from '../../googlesitekit/constants';
+import { Provider as ViewContextProvider } from '../Root/ViewContextContext';
 
 function Template() {
 	return (
 		<Layout>
```

Comment:
We shouldn't wrap these components in `Layout`, as the result is not representative of how the plugin looks. I'll apply this change myself to save a round trip.

```suggestion
```

_Story:_
![image](https://github.com/user-attachments/assets/33772976-d16a-4f42-adcc-69014cd9deaf)


_Plugin:_
![image](https://github.com/user-attachments/assets/d02e37fd-5a99-422d-a9e5-f2b9c0f17335)